src/target/riscv/riscv-013.c

```diff
@@ -552,6 +553,11 @@ static bool check_dbgbase_exists(struct target *target)
 			LOG_TARGET_ERROR(target, "Reached the end of DM chain (detected %u DMs in total).", count);
 			break;
 		}
+		if (next_dm > (1U << info->abits) - 1) {
+			LOG_TARGET_ERROR(target, "Supporting no more than %d bits address on this DMI bus. Aborting",
+					info->abits);
```

Comment:
The message seems to suggest that this is an OpenOCD limitation, however it's not. It's a HW issue. Please, take a look at my suggestion:
```suggestion
			LOG_TARGET_ERROR(target, "The address of the next Debug Module does not fit into %u bits, which is the width of the DMI bus address. This is a HW bug",
					info->abits);
```

In any case, please change the conversion specifier for `info->abits`, since it's an `unsigned int`, not `int`.

Reply:
It's a HW issue. Addressed, thanks.